tests/lib/rook/base.py

```diff
@@ -40,6 +44,52 @@ def destroy(self, skip=True):
         # TODO(jhesketh): Uninstall rook
         pass
 
+    def execute_in_ceph_toolbox(self, command):
+        if not self.toolbox_pod:
+            self.toolbox_pod = self.kubernetes.get_pod_by_app_label(
+                "rook-ceph-tools")
+
+        return self.kubernetes.execute_in_pod(command, self.toolbox_pod)
+
+    @abstractmethod
+    def preinstall(self):
+        pass
+
+    def install(self):
```

Comment:
Why is that in the base class when it's not used in the `upstream.py` implementation?

Reply:
My point was that the deployment of rook is common whether it is upstream or SES.
preinstall() would then contain the specific code (i.e rsync yaml for ses and build rook for upstream.

This PR dealing with SES I did not want to touch upstream

Reply:
Can you then just do it in the ses implementation and do it correctly in another commit/PR later?  The state with this PR is just confusing because it implements an install method in base that is not used everywhere.

Reply:
Moved everything to ses and will do another PR for these methods.